modules/job-exec/variables.tf

```diff
@@ -40,12 +40,24 @@ variable "exec" {
   default     = false
 }
 
+variable "create_service_account" {
+  description = "Create service account for the job. If false, service_account_email must be provided."
```

Comment:
> If false, service_account_email must be provided.
I believe CR Job will use default compute SA if service_account_email is not provided. We should update the description.